.gitignore

```diff
@@ -1,2 +1,4 @@
 /target
 .idea
+Cargo.lock
```

Comment:
`Cargo.lock` should be checked-in for bin projects. Please don't remove or ignore it.